src/styles/index.css

```diff
@@ -0,0 +1,49 @@
+.main-container {
+	margin-top: 7rem;
+	margin-left: 1em;
+	border-radius: 1em;
+}
+
+.videoContainer {
+	width: 98%;
+	height: 70%;
+	margin-left: 1rem;
+}
+
+.video-inner-container {
+	position: absolute;
+	padding: 2em;
+	left: 2em;
+	top: 10rem;
+	color: var(--color-pureWhite);
+}
+
+.imageContainer {
+	width: 98%;
+	height: 30em;
+}
+
+.big-container {
+}
```

Comment:
remove if not needed

Reply:
Okay I'll do that, Thank you Apurva for your detailed PR review